packages/block-editor/src/components/block-navigation/list.js

```diff
@@ -95,3 +86,43 @@ export default function BlockNavigationList( {
 		/* eslint-enable jsx-a11y/no-redundant-roles */
 	);
 }
+
+BlockNavigationList.defaultProps = {
+	selectBlock: () => {},
+};
+
+const BlockNavigationBranch = ( { withSlot, children, ...props } ) => {
```

Comment:
The code looks, it might be nice to move BlockNavigationBranch and the slot/fills into a separate file.